<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1486</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion de Trésorerie</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTreeWidget" name="arborescenceComptes">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>0</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="headerCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>57</number>
      </attribute>
      <attribute name="headerHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Compte</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Solde</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layoutCompte">
      <item>
       <layout class="QHBoxLayout" name="infosCompte">
        <item>
         <widget class="QLabel" name="labelNomCompteSelectionne">
          <property name="text">
           <string>Nom du compte</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelRapprochementCompteSelect">
          <property name="text">
           <string>Solde : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSoldeCompteSelect">
          <property name="text">
           <string>Ancien solde :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabOp">
         <attribute name="title">
          <string>Opérations</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTableWidget" name="operationsCompteSelectionne">
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="columnCount">
               <number>6</number>
              </property>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Date</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Reference</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Titre</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Debit</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Credit</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Solde</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabTr">
         <attribute name="title">
          <string>Transactions</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QWidget" name="affichageTransactions" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QTableWidget" name="tableTransactions">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="showGrid">
                <bool>false</bool>
               </property>
               <column>
                <property name="text">
                 <string>Date</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Référence</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Titre</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>ID du compte</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Nom du compte</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Débit</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Crédit</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Supprimer</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QPushButton" name="boutonValiderModif">
                 <property name="text">
                  <string>Valider les modifications</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="boutonAnnulerModif">
                 <property name="text">
                  <string>Annuler</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabHistorique">
         <attribute name="title">
          <string>Historique des rapprochements</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="textHistorique">
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="formAjouterCompteVirtuel" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QFormLayout" name="formAjoutCompte">
       <item row="0" column="0">
        <widget class="QLabel" name="labelNom">
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nom"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelType">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="type">
         <item>
          <property name="text">
           <string>Actif</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Passif</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Recettes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dépenses</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelParent">
         <property name="text">
          <string>Parent</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="valeurParent">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelVirtuel">
         <property name="text">
          <string>Virtuel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="checkBoxVirtuel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelSoldeInitial">
         <property name="text">
          <string>Solde Initial</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="valeurSoldeInitial">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>-1000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="boutonValiderCreationCompte">
           <property name="text">
            <string>Valider</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="boutonAnnulerCreationCompte">
           <property name="text">
            <string>Annuler</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="formTransactionMainWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QWidget" name="formAjouterTransaction" native="true">
         <layout class="QFormLayout" name="formAjouterTransactionLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelTitreTransaction">
            <property name="text">
             <string>Nouvelle Transaction</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelMemoTransaction">
            <property name="text">
             <string>Nom</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditMemoTransaction"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelReferenceTransaction">
            <property name="text">
             <string>Référence</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelDateTransaction">
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDateEdit" name="dateEditTransaction">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QPushButton" name="boutonValiderCreationTransaction">
            <property name="text">
             <string>Valider</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QPushButton" name="boutonAnnulerCreationTransaction">
            <property name="text">
             <string>Annuler</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditRef"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollAreaOperation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>250</width>
            <height>273</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>548</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayoutOperation">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="boutonAjouterOp">
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="boutonRetirerOp">
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1486</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionCreer_un_nouveau_projet"/>
    <addaction name="actionImporterFichier"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuComptes">
    <property name="title">
     <string>Comptes</string>
    </property>
    <addaction name="actionCreer_un_compte"/>
    <addaction name="actionSupprimer_un_compte"/>
   </widget>
   <widget class="QMenu" name="menuTransactions">
    <property name="title">
     <string>Transactions</string>
    </property>
    <addaction name="actionCreerTransaction"/>
   </widget>
   <widget class="QMenu" name="menuG_n_rer_un_document">
    <property name="title">
     <string>Générer un document</string>
    </property>
    <addaction name="actionGenererReleve"/>
    <addaction name="actionGenererBilan"/>
    <addaction name="actionGenererResultat"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuComptes"/>
   <addaction name="menuTransactions"/>
   <addaction name="menuG_n_rer_un_document"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSauvegarder"/>
   <addaction name="actionImporterFichier"/>
   <addaction name="actionCreer_un_nouveau_projet"/>
   <addaction name="separator"/>
   <addaction name="actionCreer_un_compte"/>
   <addaction name="actionSupprimer_un_compte"/>
   <addaction name="actionRapprocher"/>
   <addaction name="separator"/>
   <addaction name="actionCreerTransaction"/>
   <addaction name="actionCloture"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAfficherTransactions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/afficher_transactions.png</normaloff>../icons/afficher_transactions.png</iconset>
   </property>
   <property name="text">
    <string>Afficher les transactions</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCreerTransaction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/nouvelle_transaction.png</normaloff>../icons/nouvelle_transaction.png</iconset>
   </property>
   <property name="text">
    <string>Créer une nouvelle transaction</string>
   </property>
  </action>
  <action name="actionRapprocher">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/rapprocher_compte.png</normaloff>../icons/rapprocher_compte.png</iconset>
   </property>
   <property name="text">
    <string>Rapprocher le compte</string>
   </property>
  </action>
  <action name="actionImporterFichier">
   <property name="icon">
    <iconset>
     <normaloff>../icons/importer_projet.png</normaloff>../icons/importer_projet.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir...</string>
   </property>
  </action>
  <action name="actionCreer_un_compte">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/nouveau_compte.png</normaloff>../icons/nouveau_compte.png</iconset>
   </property>
   <property name="text">
    <string>Créer un compte</string>
   </property>
  </action>
  <action name="actionSupprimer_un_compte">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/supprimer_compte.png</normaloff>../icons/supprimer_compte.png</iconset>
   </property>
   <property name="text">
    <string>Supprimer le compte sélectionné</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/save.png</normaloff>../icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCloture">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/cloture_livre.png</normaloff>../icons/cloture_livre.png</iconset>
   </property>
   <property name="text">
    <string>Cloture</string>
   </property>
   <property name="toolTip">
    <string>Clôture du livre</string>
   </property>
  </action>
  <action name="actionCreer_un_nouveau_projet">
   <property name="icon">
    <iconset>
     <normaloff>../icons/nouveau_projet.png</normaloff>../icons/nouveau_projet.png</iconset>
   </property>
   <property name="text">
    <string>Créer un nouveau projet</string>
   </property>
  </action>
  <action name="actionGenererReleve">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/generer_doc.png</normaloff>../icons/generer_doc.png</iconset>
   </property>
   <property name="text">
    <string>Générer un relevé</string>
   </property>
  </action>
  <action name="actionGenererBilan">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/generer_doc.png</normaloff>../icons/generer_doc.png</iconset>
   </property>
   <property name="text">
    <string>Générer un bilan</string>
   </property>
  </action>
  <action name="actionGenererResultat">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/generer_doc.png</normaloff>../icons/generer_doc.png</iconset>
   </property>
   <property name="text">
    <string>Générer le résultat</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionCreer_un_compte</sender>
   <signal>triggered()</signal>
   <receiver>formAjouterCompteVirtuel</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boutonAnnulerCreationCompte</sender>
   <signal>clicked()</signal>
   <receiver>formAjouterCompteVirtuel</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boutonAnnulerCreationTransaction</sender>
   <signal>clicked()</signal>
   <receiver>formTransactionMainWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>621</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>825</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCreer_un_compte</sender>
   <signal>triggered()</signal>
   <receiver>formTransactionMainWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>810</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
